yt/docs/README.md

```diff
@@ -1,208 +1,434 @@
+               |-- dyn-query-language.md
+                   
+```
 
-Права на создание и удаление групп в данный момент не выдаются пользователям и осуществляются через форму или вкладку ACL узла в веб-интерфейсе. Добавление/удаление пользователей в группы делегированы ответственным через IDM.
+Text of this article is included from the `en/_includes/user-guide/query-tracker.md` file:
 
-{% endnote %}
+```
+{% include [Query Tracker](../_includes/user-guide/query-tracker.md) %}
 ```
 
-Предупреждающие примечания задаются: `{% note warning %} ... {% endnote %}`. Само примечание начинается с большой буквы.
+Now, open the `en/_includes/user-guide/query-tracker.md` file. There you can find the link to the "YT Query Language" article:
 
 ```
-{% note info "Примечание" %}
+Currently supported execution engines include: [YT QL](../../user-guide/dynamic-tables/dyn-query-language.md).
+```
+
+The `dyn-query-language.md` article also reuses content, same as `query-tracker.md`. So, as you can see in the code above:
+- The "YT QL" link refers to the article located in the `en/user-guide/dynamic-tables/` folder, but not the `en/_includes/user-guide/dynamic-tables/`.
+- This link is set relativelly to the `_includes` folder.
 
-Пользователи и группы располагаются в одном пространстве имен, то есть имена пользователей и групп не могут совпадать.
+### Headers
 
-{% endnote %}
+Headers should always have anchors:
+
+```
+## Users, Groups { #users-groups }
+```
+
+
+### Lists
+
+Every item in an ordered list starts with a capital letter and ends with a period, while items in an unordered list start with a lowercase letter and end with a semicolon; the last item in the list always ends with a period.
+
+**Ordered:**
+
+```
+1. The type of access (read, write, etc.).
+2. The user who initiated the request.
+3. The object to which access is requested.
 ```
 
-Если для понимания раздела есть требования, то они оформляются следующим образом:
+**Unordered:**
 
 ```
-{% note warning "Внимание" %}
+- the type of access (read, write, etc.);
+- the user who initiated the request;
+- the object to which access is requested.
+```
+
+### Code Examples Snippets
+
+**Bash**
+
+In the bash examples, add the `$` symbol at the beginning of the line before the command. This allows to visually distinguish the command invocation from the output response.
 
-Для понимания изложенного в разделе материала требуется знание основ/устройства/принципов ... того-то (со ссылками)
+Example:
 
-{% endnote %}
+````
+```bash
+$ yt get //home/@count
+36
 ```
+````
+
+### Tables
+
+When working with tables, keep the following in mind:
 
-## Таблицы
+- There must be a reference to the table, for example: "Table 1 shows...".
+- It is mandatory to add a table title: `<small>Table N — Title of the table`. It should start with a capital letter, without a period at the end. 
+- Periods are not needed in the table header, as well as at the end of the sentence in the right column.
```

Comment:
Does it mean periods are mandatory at the end of the sentence in the non-right column?
The example doesn't answer it since `name` or `string` are not sentences